src/jsonrpc/miner_jsonrpc_info.erl

```diff
@@ -31,7 +31,15 @@ handle_rpc(<<"info_name">>, []) ->
 handle_rpc(<<"info_block_age">>, []) ->
     #{block_age => miner:block_age()};
 handle_rpc(<<"info_p2p_status">>, []) ->
-    maps:from_list(miner:p2p_status());
+    [{"connected", Connected}, {"dialable", Dialable}, {"nat_type", NatType}, {"height", Height}] = miner:p2p_status(),
```

Comment:
This patch will break if a new element is added or the ordering of the list changes. . Match on the result from `maps:from_list` to avoid changes to p2p_status output. 

Try something like 

```
#{
    "connected" := Connected,
    "dialable" := Dialable,
... 
} = maps:from_list(miner:p2p_status())
```

Reply:
let me know what you think about the most recent change, I built and tested the results against mainnet, seems to be working well, an example response has been included below.

Reply:
I ended up removing the wrapping p2p_status record, it just feels like overhead, and there isn't 100% consistency between the other calls, but I'd prefer no wrapper when using the results.